src/nswcaselaw/decision.py

```diff
@@ -47,6 +45,8 @@
     "category",
     "fileNumber",
     "representation",
+    "courtOrTribunal",
+    "citation",
```

Comment:
Are these two fields for the decision under review? I'm not sure that they're being populated, as the decision under review is being stored in a dict. We might need some extra code to get the decision under review values into CSV, and give them column names which make it clear that it's the review decision metadata

Reply:
I think I mistakenly added these field at the beginning, maybe we can remove them and a new decisionUnderAppeal field to the `CSV_FIELDS` variable?